### ISSUE TITLE
U2C needed with M8P ?

### ISSUE THREAD

**Issue by Kostt3d** [bigtreetech/Manta-M8P#75]
Hi,
u2c is needed to configure the ebb42 can with manta m8p? i can plug the ebb to my laptop with usb c and finish install with port can on the manta ?

**Comment by rbentoski2113** [comment]
yes for version 1.0 m8p at least, thats what i got. the 1.1 has a can header in the middle of the board.   the u 2 c is a pretty straight foward set up, highly recomend going with canboot.  currently my set up is m8p w cb1, u2c, with the fystec PITB(double 5160 can board) on one port and EBB36 on the other and it was simple to set up. i belive if i wanted to i can add like 2 more , also good thing about can is you can daisy chain if they have the ports on it, e.g PITB board
